### ISSUE TITLE
Fast fail on missing tools

### ISSUE THREAD

**Issue by AVee** [kivy/buildozer#1006]
### Versions

* Python: 3.7
* OS: Windows 10 + WSL + Debian Buster
* Buildozer: 1.0

### Description

Buildozer does a bunch of checks for the presence of required programs when starting but fails to check for the presence of a unzip command:
```# Unpacking Android SDK
# Run 'unzip -q /home/avee/.buildozer/android/platform/android-sdk/sdk-tools-linux-4333796.zip'
# Cwd /home/avee/.buildozer/android/platform/android-sdk
# Command failed: unzip -q /home/avee/.buildozer/android/platform/android-sdk/sdk-tools-linux-4333796.zip
...
(env) avee@localhost:~$ unzip
-bash: unzip: command not found
```
Obviously that's easily fixed by installing unzip, but it would be nice if buildozer would fail early. 

For some missing tools there are warnings:
```[WARNING]: Missing executable: pkg-config is not installed
[WARNING]: Missing executable: autoconf is not installed
[WARNING]: Missing executable: automake is not installed
[WARNING]: Missing executable: libtoolize is not installed
```
It would be nice if those would fail as well, at least in a default run (I can see the use of something like an --ignore-missing-executables flag). Failing fast on relatively trivial things like this can save a lot of time when setting up a build environment (especially when doing it for the first time).

Command:
```buildozer android debug```

**Comment by AVee** [comment]
And the same goes for the `zip` command. (Only finding that out a 2 hours later illustrates the point...)

**Comment by AndreMiras** [comment]
Yes that would be nice, not only with executables, but also other build dependencies e.g. libraries.
Pull requests are welcome

**Comment by Julian-O** [comment]
Note: Do not perform this check when running Windows. `unzip` isn't used in that case (even though Windows support isn't there yet)

`buildops.checkbin()` makes this an easy feature to add.
